content/en/docs/setup/platform-setup/tencent-cloud-mesh/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Tencent Cloud
+description: Instructions to setup Istio quickly in Tencent Cloud.
+weight: 65
+skip_seealso: true
+keywords: [platform-setup,tencent-cloud-mesh,tcm,tencent-cloud,tencentcloud]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+## Prerequisites
+
+Follow these instructions to prepare a [Tencent Kubernetes Engine](https://intl.cloud.tencent.com/products/tke) or [Elastic Kubernetes Service](https://intl.cloud.tencent.com/product/eks) cluster for Istio.
+
+You can deploy a Kubernetes cluster to Tencent Cloud via [Tencent Kubernetes Engine](https://intl.cloud.tencent.com/document/product/457/40029) or [Elastic Kubernetes Service](https://intl.cloud.tencent.com/document/product/457/34048) which fully supports Istio.
+
+{{< image link="./tke.png" caption="Create Cluster" >}}
+
+## Procedure
+
+After creating a Tencent Kubernetes Engine or Elastic Kubernetes Service cluster, you can quickly start to deploy and use Istio by [Tencent Cloud Mesh](https://cloud.tencent.com/product/tcm):
+
+{{< image link="./tcm.png" caption="Create Tencent Cloud Mesh" >}}
+
+1. Log on to the `Container Service console`, and click **Service Mesh** in the left-side navigation pane to enter the **Service Mesh** page.
+
+1. Click the **Create** button in the upper-left corner.
+
+1. Enter the mesh name.
+
+    {{< tip >}}
+    The mesh name can be 1–60 characters long and it can contain numbers, Chinese characters, English letters, and hyphens (-).
+    {{< /tip >}}
+
+1. Select the **Region** and **Zone** in which the cluster resides.
+
+1. Choose the Istio version. Tencent Cloud Mesh supports the latest two major versions of Istio.
+
+1. Choose the service mesh mode: `Standalone` or `Hosted`.
```

Comment:
The image has **Managed Mesh** and **Stand-Alone Mesh**. Maybe change Hosted to Managed in the doc.

Reply:
Updated @ericvn PTAL.